source/documentation/deploying_services/sqs.md

```diff
@@ -0,0 +1,342 @@
+# Amazon SQS
+
+Amazon SQS provides "standard" (at-least-once delivery, best-effort ordering) and first-in-first-out queues.
```

Comment:
Would be good to provide links for all these concepts

Reply:
These aren't really concepts that have much follow up information, however I have removed them from this section so now they only appear in the section that links off to the AWS docs, so that can hopefully serve for more of a deep dive into the different types.